src/test/java/org/opensearch/plugin/insights/core/exporter/LocalIndexExporterTests.java

```diff
@@ -95,4 +97,14 @@ public void testGetAndSetIndexPattern() {
         localIndexExporter.setIndexPattern(newFormatter);
         assert (localIndexExporter.getIndexPattern() == newFormatter);
     }
+
```

Comment:
As I mentioned above, we should add more unit tests for deleting the indices here.

Reply:
Added